xc/common/cmake/arch_define.cmake

```diff
@@ -65,7 +65,7 @@ function(ADD_XC_ARCH_DEFINE)
       --router_heap bucket \
       --clock_modeling route \
       --place_delta_delay_matrix_calculation_method dijkstra \
-      --place_delay_model delta_override \
+      --place_delay_model delta \
```

Comment:
Given that the results look good, this is fine, but this a little surprising.  We should talk to Vaughn about this.

Reply:
Ok with me, given discussion today.